OWML.Common/CommandLineArguments.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public static class CommandLineArguments
+    {
+        public static string GetArgument(string name)
+        {
+            var arguments = Environment.GetCommandLineArgs();
+            for (var i = 0; i < arguments.Length; i++)
+            {
+                var argument = arguments[i];
+                if (argument == $"-{name}" && arguments.Length > i)
+                {
+                    return arguments[i + 1];
+                }
+            }
+            return null;
```

Comment:
`var argument = arguments.FirstOrDefault(arg => arg == $"-{name}");
return string.IsNullOrEmpty(argument) ? null : arguments[arguments.IndexOf(nameArgument) + 1];`
Yours is a bit more robust, your call.

Reply:
I like yours 👍

Reply:
I simplified it but I preferred to keep it a bit more readable, hope it looks ok.